docs/docs/widgets/divider.md

```diff
@@ -50,10 +50,12 @@ Under the <b>General</b> accordion, you can set the value in the string format.
 
 ## Styles
 
-| <div style={{ width:"100px"}}> Style </div> | <div style={{ width:"100px"}}> Description </div>                                               |
-| ------------------------------------------- | ----------------------------------------------------------------------------------------------- |
-| Divider Color                               | It is used to set the color of the divider. Use hex code to set the background color.           |
-| Visibility                                  | This property is used to set the visibility of the divider. The property accepts Boolean value. |
+| <div style="width:100px;">Property</div>    | <div style="width:100px;">Description</div> | <div style="width:100px;">Configuration Options</div>  |
+| ------------------------------------------- | ------------------------------------------- | -------------------------------------------------------|
+| Divider Color    | Sets the background color of the divider. The value must be provided as a HEX color code (e.g., #FFFFFF).   | You can directly input the HEX code or click on *fx* to programmatically configure the background color.    |
```

Comment:
Here, for the divider color's description, can you please remove the second sentence `The value must be provided as a HEX color code (e.g., #FFFFFF).`. Also for the configuration option can you please use the content from the [reference](https://docs.tooljet.com/docs/widgets/text-input/#field)?